app/views/catalog/_index_hierarchy_default.html.erb

```diff
@@ -1,4 +1,5 @@
 <%= content_tag('div', class: 'al-document-abstract-or-scope', title: document.abstract_or_scope) do %>
+  <%= content_tag(:h4,'Scope and Contents', class: 'al-document-abstract-or-scope-heading') %>
```

Comment:
Can we internationalize this string?  This will allow downstream applications to more easily control this label in the UI.